core/block_svg.js

```diff
@@ -965,6 +977,11 @@ Blockly.BlockSvg.prototype.onMouseMove_ = function(e) {
       }
       this.setDragging_(true);
       this.moveToDragSurface_();
+
+
+      if (!Blockly.draggedConnectionManager_) {
```

Comment:
It might still make the rest of the refactor easier if you set up your control and data classes for drags now.

Maybe something like:
Blockly.gestureHandler.startDrag(this);

startDrag(block) {
  if (!this.draggingConnectionManager_)
    this.draggingConnectionManager_ = new Blockly.DraggingConnectionManager(block);
  }
}

Then you can start pulling more logic out of BlockSVG and Flyout